src/gui/properties/propertieswidget.cpp

```diff
@@ -193,24 +193,31 @@ void PropertiesWidget::showPiecesDownloaded(bool show)
 
 void PropertiesWidget::setVisibility(bool visible)
 {
+    static int handleWidth = 0; // not 5 everywhere
```

Comment:
leave this change out of this PR, it is unrelated.

Reply:
Sorry but this is related to this PR, the extra top margin in collapsed properties widget (no tab is visible) is due to the disabled splitter handle, actually even when we disable QSplitter::handle, it still is visible (takes space), so I make `QSplitter::handleWidth` (here it is height because of the vertical arrangement of splitter) = 0, when we disable the QSplitter::handle